UpTraceNuxt/nuxt.config.js

```diff
@@ -40,12 +41,49 @@ export default {
 
   // Modules: https://go.nuxtjs.dev/config-modules
   modules: [
-    // https://go.nuxtjs.dev/axios
-    '@nuxtjs/axios',
+    [
+      '@nuxtjs/axios',
+      '@nuxtjs/firebase',
+      {
+        config: {
+          apiKey: 'AIzaSyDeG44ayhy1YyMV6rkOU3lsNdbkElPYJks',
+          authDomain: 'attempt2-302520.firebaseapp.com',
+          databaseURL: 'attempt2-302520.firebaseio.com',
+          projectId: 'attempt2-302520',
+          storageBucket: "attempt2-302520.appspot.com",
+          messagingSenderId: "635480116758",
+          appId: "1:635480116758:web:23a113f952699f80882f37",
+          measurementId: "G-HNCPYRMMTP"
```

Comment:
creds will need to be removed from the repo and moved to some sort of secret store eventually (don't worry about it for now)